plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/CodeQLReader.java

```diff
@@ -190,75 +190,11 @@ private TestCaseResult parseLGTMFinding(
-                 * for them in Benchmark currently. They are left here in case we do see them in the
-                 * future to make it easier to support them.
-                // These rules we care about, but have to map to the CWE we expect
-                    case 338: // java/jhipster-prng
-                        return 330; // Weak Random
-                    case 347: // java/missing-jwt-signature-check - TODO - Does this affect score?
-                        return 327; // Weak Crypto
-
-                    // These rules we don't care about now, but we return their CWE value anyway in case
-                    // we care in the future
-                    case 94: // java/insecure-bean-validation and many others
-                    case 190: // java/implicit-cast-in-compound-assignment
-                    case 197: // java/tainted-numeric-cast
-                    case 297: // java/unsafe-hostname-verification
-                    case 300: // java/maven/non-https-url
-                    case 315: // java/cleartext-storage-in-cookie
-                    case 352: // java/spring-disabled-csrf-protection
-                    case 502: // java/unsafe-deserialization
-                    case 601: // java/unvalidated-url-redirection
-                    case 732: // java/world-writable-file-read
-                    case 807: // java/tainted-permissions-check
-                    case 917: // java/ognl-injection
-                    case 918: // java/ssrf
-                    case 1104: // java/maven/dependency-upon-bintray
-                */
-
-            case 113: // java/http-response-splitting
-            case 117: // js/log-injection
-            case 134: // java/tainted-format-string
-            case 209: // java/stack-trace-exposure
-            case 404: // java/database-resource-leak
-            case 477: // java/deprecated-call
-            case 485: // java/abstract-to-concrete-cast
-            case 561: // java/unused-parameter
-            case 563: // js/useless-assignment-to-local
-            case 570: // java/constant-comparison
-            case 685: // java/unused-format-argument
-            case 730: // js/regex-injection (i.e., DOS)
-            case 776: // js/xml-bomb (i.e., XEE, as opposed to XXE, which is already mapped above
-            case 843: // js/type-confusion-through-parameter-tampering
-                return cweNumber.intValue(); // Return CWE as is
-            default:
-                System.out.println(
-                        "CodeQL parser encountered new unmapped vulnerability type: "
-                                + cweNumber
-                                + " for rule: "
-                                + ruleName);
+    private CweNumber mapCWE(Integer cweNumber) {
+        if (cweNumber == 335) {
+            return CweNumber.WEAK_RANDOM;
         }
-        return 0; // Not mapped to anything
+
+        return CweNumber.lookup(cweNumber);
```

Comment:
I prefer that we keep the manual mapping so we can get the warning of 'new' types of issues not yet mapped.

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379